components/molecule/tabs/src/styles/settings.scss

```diff
@@ -1 +1,33 @@
-@import '../_settings.scss';
```

Comment:
this removal is not retrocompatible cuz there are someone importing ancient file for tokens.

Reply:
Keep old token declaration file path until we make a major version

Reply:
my bad! I didn't think it was going to break!